serv/semrider.py

```diff
@@ -1,15 +1,19 @@
 import serv.iops as iops
 import serv.algo as alg
+import os
```

Comment:
Put 'os' first, all system level imports first and our imports last. That way, we we override with our custom it works (see Google Style Guide on Python).